messaging/coap/engine.c

```diff
@@ -924,14 +910,6 @@ coap_receive(oc_message_t *msg)
     }
   }
 
-#ifdef OC_SECURITY
-  if (coap_status_code == CLOSE_ALL_TLS_SESSIONS) {
```

Comment:
This was redundant?

Reply:
Yes. it was not obvious when the tls connections will be closed. - refactored